docs/machine-learning/how-to-guides/load-data-ml-net.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "How-To: Load Data in ML.NET"
+description: Load data file and streaming data into ML.NET
+ms.date: 04/03/2019
+ms.custom: mvc,how-to
+#Customer intent: As a developer I want to know how to load data from file and streaming data sources.
+---
+
+# How-To: Load Data in ML.NET
+
+This how-to shows you how to load data for processing and training into ML.NET that's stored in files or real-time / streaming data sources.
+
+## Create Data Model Schema
+
+ML.NET allows for defining data model schemas via classes. For example, given the following input data:
+
+```text
+Size (Sq. ft.), HistoricalPrice1 ($), HistoricalPrice2 ($), HistoricalPrice3 ($), Current Price ($)
+700, 100000, 3000000, 250000, 500000
+1000, 600000, 400000, 650000, 700000
+```
+
+Create a data model that looks like the snippet below:
+
+```csharp
+class HousingData
+{
+    [LoadColumn(0)]
+    public float Size { get; set; }
+    [LoadColumn(1,3)]
+    [VectorType(3)]
+    public float[] HistoricalPrices { get; set; }
+    [LoadColumn(4)]
+    public float CurrentPrice { get; set; }
+}
+```
+
+### Column Attributes
```

Comment:
I'd leave it as Attributes because that is how they are defined in .NET. Annotations seems nonstandard.  https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/attributes/

Reply:
Ok, sounds good!